app/models/service_template_ansible_playbook.rb

```diff
@@ -117,4 +109,25 @@ def self.validate_config_info(info)
     info
   end
   private_class_method :validate_config_info
+
+  def job_template(action)
+    resource_actions.find_by!(:action => action.to_s.capitalize).configuration_template
+  end
+
+  def update_catalog_item(options, auth_user = nil)
+    config_info = validate_update_config_info(options)
```

Comment:
Base's `validate_update_config_info` is not sufficient, we need to override it like this

```
def validate_update_config_info(options)`
  opts = super
  self.class.send(:validate_config_info, opts)
end
```